docs/en/index.md

```diff
@@ -11,117 +11,117 @@ Last released to Markdown: not yet released
 | Minimum requirements | 500 MHz single core processor256 MB RAM512 MB storage                                                    | 1 GHz dual core processor512 MB RAM1.5 GB storage                                                                                                              |
 | Graphical UI         | None by default (can utilize Wayland or Mir)                                                             | Xorg and GNOME Shell or Wayland and GNOME Shell                                                                                                                |
 | Package system       | snaps                                                                                                    | Debs & snaps                                                                                                                                                   |
 | Application security | Isolation via AppArmor and Seccomp                                                                       | Traditional user and group permissions (for Debs);Strict isolation via AppArmor and Seccomp for snaps;Transitional security with classic confinement for snaps |
-| Updates              | Pushed from the Global (public) store & optionally a Brand (private) store;All updates are transactional | Traditional apt repository updates;Also from public store & optionally Brand store for snaps;Updates for snaps are transactional                               |
+| Updates              | Pushed from the Global (public) Store & optionally a Brand (private) Store;All updates are transactional | Traditional apt repository updates;Also from public Store & optionally Brand Store for snaps;Updates for snaps are transactional                               |
 
 
 ## Snaps
 
-Snap is the native Ubuntu Core packaging system. There is no Debian package support in the runtime of Ubuntu Core. And, Snaps are increasingly used in classic Ubuntu flavours and other Linux distributions. 
+Snap is the native Ubuntu Core packaging system. There is no Debian package support in the runtime of Ubuntu Core. And, Snaps are increasingly used in Ubuntu Classic flavours and other Linux distributions. 
 
 Snaps solve multiple problems at the same time:
 
  - It’s easy to package applications pulled from multiple upstream sources using diverse build systems.
 Each snap includes its own dependencies, freeing publishers from problems when  implicit dependencies change without their knowledge.
- - Developers publish snaps in a snap store (the global snap store or a private Brand store) and take responsibility for their quality, making software distribution fast and free of procedural obstacles.
+ - Developers publish snaps in a Snap Store (the Global Snap Store or a public or private Brand Store) and take responsibility for their quality, making software distribution fast and free of procedural obstacles.
```

Comment:
it could mean the store is Listed (so everyone sees it) and anyone can publish snaps there (not just Publishers). Public/private on brand store is not well defined so actually I'd stay away from those terms.

Reply:
Yes agreed.